azure-iot-hub-devicesdk/samples/legacy-samples/README.md

```diff
@@ -0,0 +1,28 @@
+# Legacy Scenario Samples for the Azure IoT Hub Device SDK
```

Comment:
I put both Device/Hub and Module/Edge samples in this directory because I didn't want two different legacy directories. We can adjust this if anyone feels strongly about it.